src/plugins/gui/pluginUtils.ts

```diff
@@ -48,32 +39,53 @@ export const getBestError = (errorQuotes: FiatProviderError[], currencyCode: str
       bestError = errorQuote
       continue
     }
-    // @ts-expect-error
     if (ERROR_PRIORITIES[errorQuote.errorType] < ERROR_PRIORITIES[bestError.errorType]) {
       bestError = errorQuote
       continue
     }
-    // @ts-expect-error
     if (ERROR_PRIORITIES[errorQuote.errorType] === ERROR_PRIORITIES[bestError.errorType]) {
       if (errorQuote.errorType === 'overLimit' && bestError.errorType === 'overLimit') {
-        if (errorQuote.errorAmount > bestError.errorAmount) {
+        if (bestError.errorAmount != null && errorQuote.errorAmount != null && errorQuote.errorAmount > bestError.errorAmount) {
           bestError = errorQuote
         }
       } else if (errorQuote.errorType === 'underLimit' && bestError.errorType === 'underLimit') {
-        if (errorQuote.errorAmount < bestError.errorAmount) {
+        if (bestError.errorAmount != null && errorQuote.errorAmount != null && errorQuote.errorAmount < bestError.errorAmount) {
```

Comment:
Change to

`if ((errorQuote.errorAmount ?? Infinity) < (bestError.errorAmount ?? Infinity)) {`